app/src/main/java/com/trantan/music53/ui/info_track/InfoTrackFragment.java

```diff
@@ -0,0 +1,84 @@
+package com.trantan.music53.ui.info_track;
+
+import android.os.Bundle;
+import android.support.design.widget.BottomSheetDialogFragment;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.load.resource.bitmap.RoundedCorners;
+import com.bumptech.glide.request.RequestOptions;
+import com.trantan.music53.R;
+import com.trantan.music53.data.Track;
+import com.trantan.music53.utils.StringUtil;
+
+/**
+ * A simple {@link Fragment} subclass.
+ */
+public class InfoTrackFragment extends BottomSheetDialogFragment {
+    private static final String ARGUMENT_TRACK = "ARGUMENT_TRACK";
+    private static final int INT_TEN = 10;
+    private ImageView mImageArtwork;
+    private TextView mTextTitle;
+    private TextView mTextArtist;
+    private TextView mTextComment;
+    private TextView mTextPlay;
+    private TextView mTextLike;
+    private Track mTrack;
+    private View mView;
+    private TextView mTextUploadDate;
+
+    public static InfoTrackFragment getIntance(Track track) {
+        InfoTrackFragment infoTrackFragment = new InfoTrackFragment();
+        Bundle bundle = new Bundle();
+        bundle.putSerializable(ARGUMENT_TRACK, track);
```

Comment:
em thử tìm hiểu và so sánh giuwaax serializable and parcelable

Reply:
Vâng ạ